x-pack/plugins/observability/public/components/app/section/alerts/index.tsx

```diff
@@ -49,92 +51,91 @@ export function AlertsSection({ alerts }: Props) {
-                compressed
-                id="filterAlerts"
-                options={[allTypes, ...filterOptions]}
-                value={filter}
-                onChange={(e) => setFilter(e.target.value)}
-                prepend={i18n.translate('xpack.observability.overview.alert.view', {
-                  defaultMessage: 'View',
+    <EuiFlexGroup direction="column" gutterSize="none">
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup justifyContent="flexEnd">
+          <EuiFlexItem>
+            <EuiTitle size="xs">
+              <h4>
+                {i18n.translate('xpack.observability.overview.alerts.title', {
+                  defaultMessage: 'Alerts',
                 })}
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </EuiFlexItem>
-        <EuiHorizontalRule margin="xs" />
-        <EuiFlexItem>
-          {alerts
-            .filter((alert) => filter === ALL_TYPES || alert.consumer === filter)
-            .map((alert, index) => {
-              const isLastElement = index === alerts.length - 1;
-              return (
-                <EuiFlexGroup direction="column" gutterSize="s" key={alert.id}>
-                  <EuiSpacer size="s" />
-                  <EuiFlexItem>
-                    <EuiLink
-                      href={core.http.basePath.prepend(
-                        `/app/management/insightsAndAlerting/triggersActions/alert/${alert.id}`
-                      )}
-                    >
-                      <EuiText size="s">{alert.name}</EuiText>
-                    </EuiLink>
-                  </EuiFlexItem>
-                  <EuiFlexItem grow={false}>
-                    <EuiFlexGroup gutterSize="xs">
-                      <EuiFlexItem grow={false}>
-                        <EuiBadge color="hollow">{alert.alertTypeId}</EuiBadge>
-                      </EuiFlexItem>
-                      {alert.tags.map((tag, idx) => {
-                        return (
-                          <EuiFlexItem key={idx} grow={false}>
-                            <EuiBadge color="default">{tag}</EuiBadge>
-                          </EuiFlexItem>
-                        );
-                      })}
-                    </EuiFlexGroup>
-                  </EuiFlexItem>
-                  <EuiFlexItem>
-                    <EuiFlexGroup gutterSize="s">
+              </h4>
+            </EuiTitle>
+            <EuiSpacer size="m" />
+            <EuiSelect
+              compressed
+              id="filterAlerts"
+              options={[allTypes, ...filterOptions]}
+              value={filter}
+              onChange={(e) => setFilter(e.target.value)}
+              prepend={i18n.translate('xpack.observability.overview.alert.view', {
+                defaultMessage: 'View',
+              })}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButton size="s" href="xpack.observability.overview.alert.appLink">
```

Comment:
I wasn't able to get this button working. The previous code was this `appLink={{
        href,
        label: i18n.translate('xpack.observability.overview.alert.appLink', {
          defaultMessage: 'Manage alerts',
        }),
      }}`

Reply:
@katefarrar it should be 
```suggestion
            <EuiButton size="s" href={href}>
              {i18n.translate('xpack.observability.overview.alert.appLink', {
                defaultMessage: 'Manage alerts',
              })}
            </EuiButton>
```